Source/Plugins/Core/com.equella.core/src/com/tle/web/wizard/viewitem/actions/NewVersionSection.java

```diff
@@ -48,6 +56,18 @@ protected void execute(SectionInfo info) throws Exception
 		forwardToWizard(info, false, false, true);
 	}
 
+	@Override
+	public SectionResult renderHtml(RenderEventContext context)
+	{
+		Item item = getItemInfo(context).getItem();
+		if (item.isModerating())
+		{
+//		Display a warning when creating a new version of this item because there is a current version awaiting moderation.
```

Comment:
The comment marker (`//`) should align with the following line (i.e. `getComponent()...`) and then we only need one space after it.

```suggestion
			// Display a warning when creating a new version of this item because there is a current version awaiting moderation.
```